mayavi/tests/test_mlab_integration.py

```diff
@@ -211,29 +211,29 @@ def test_colorbar(self):
 ################################################################################
 class TestMlabPipeline(TestMlabNullEngine):
     """ Test the pipeline functions.
-        For vtk versions greater than 5.6 (5.10.1 onwards), widgets need
+        For vtk versions greater than 5.10.1, widgets need
```

Comment:
I agree.  So perhaps `5.10 onwards` is more true to the implementation in the code.